voluncheer/voluncheer/settings.py

```diff
@@ -153,34 +155,34 @@
 CRISPY_ALLOWED_TEMPLATE_PACKS = "bootstrap5"
 CRISPY_TEMPLATE_PACK = "bootstrap5"
 
-# Backend Email (testing)
-# if os.getenv("IS_PRODUCTION"):
-EMAIL_BACKEND = "django_ses.SESBackend"
-AWS_ACCESS_KEY_ID = os.getenv("AWS_SES_ACCESS_KEY")
-AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SES_SECRET_ACCESS_KEY")
-AWS_SES_REGION_NAME = os.getenv("AWS_SES_REGION_NAME")
-AWS_SES_REGION_ENDPOINT = os.getenv("AWS_SES_REGION_ENDPOINT")
-AWS_SES_DOMAIN = os.getenv("AWS_SES_DOMAIN")
-DEFAULT_FROM_EMAIL = os.getenv("AWS_SES_FROM_EMAIL")
-# else:
-#     EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+# Backend Email
+if not environment.is_local:
+    EMAIL_BACKEND = "django_ses.SESBackend"
+    AWS_ACCESS_KEY_ID = os.getenv("AWS_SES_ACCESS_KEY")
+    AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SES_SECRET_ACCESS_KEY")
+    AWS_SES_REGION_NAME = os.getenv("AWS_SES_REGION_NAME")
+    AWS_SES_REGION_ENDPOINT = os.getenv("AWS_SES_REGION_ENDPOINT")
+    AWS_SES_DOMAIN = os.getenv("AWS_SES_DOMAIN")
+    DEFAULT_FROM_EMAIL = os.getenv("AWS_SES_FROM_EMAIL")
+else:
+    EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
```

Comment:
Since the test is failing for import, we can just use placeholder values like "N/A".

Reply:
An empty string is as useful as nonsense so I left it empty. We can't really test email locally right now anyway.